_posts/2024-03-14-ugm2024-updates.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: UGM 2024 Updates - Keynote Speaker Announcement and Abstract Deadline Extension
+---
+
+There are exciting updates to announce regarding the [2024 MDAnalysis UGM](https://www.mdanalysis.org/pages/ugm2024/) (User Group Meeting), taking place August 21-23, 2024 in London, UK. Keep reading to learn more about the exciting talks you can expect from experts in the molecular dynamics simulations space, and a deadline extension to make sure you have time to submit your own abstract for giving a talk or presenting a poster during the UGM.
+
+# Keynote Speakers
+We are proud to introduce two fantastic keynote speakers for the UGM: Dr. Antonia Mey and Dr. Francesca Stanzione! Read more about them and their work below.
+
+## Dr. Antonia Mey
+
+<img
+src="/public/images/UGM 2024 Speaker Announcement Cards - Mey.png"
+title="Dr. Antonia Mey" alt="Dr. Antonia Mey"
+style="float: left; " />
+
+## Dr. Francesca Stanzione
+
+<img
+src="/public/images/UGM 2024 Speaker Announcement Cards - Stanzione.png"
+title="Dr. Francesca Stanzione" alt="Dr. Francesca Stanzione"
+style="float: left; " />
```

Comment:
Add links to the sections in the UGM page where there's more information about speakers.

Reply:
Now I see that the images have text. 

Then add ALL the text from the image in the alt attribute so that the information is _accessible_.

Reply:
Thanks for this suggestion!